src/plugins/charts/public/static/components/endzones.tsx

```diff
@@ -142,7 +142,7 @@ export const getAdjustedInterval = (
 
 const partialDataText = i18n.translate('charts.partialData.bucketTooltipText', {
   defaultMessage:
-    'The selected time range does not include this entire bucket, it may contain partial data.',
```

Comment:
@nickofthyme let's also delete it from the translation files in order to be translated again

Reply:
Thanks, See 1e750f0b1ea061307947082ab1bd53c1577ec0bd